copyright/copyright.js

```diff
@@ -0,0 +1,106 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+
+const copyright = {
+    items: [],
+    missingHeaders: [],
+    copyrightText: '',
+    lintMode: false,
+    updateHeaders() {
+        const filesRead = this.items.map((item) => {
+            return new Promise((resolve) => {
+                fs.readFile(item, (err, data) => {
+                    if (!this.hasCopyrightHeader(data)) {
+                        if (this.lintMode === true) {
+                            console.log(`\x1b[33m Missing copyright headers in \x1b[36m ${item}`)
+                            this.missingHeaders.push(item)
+                        } else {
+                            console.log(`\x1b[32mCopyright header succesfully written into \x1b[35m${item}`)
+                            this.writeHeader(item, data)
+                        }
+                    }
+                    resolve()
+                })
+            })
+        })
+
+        Promise.all(filesRead).then(() => {
+            if (this.lintMode === true) {
+                if (this.missingHeaders.length > 0) {
+                    console.log('\x1b[31m \x1b[40mERROR\x1b[49m - Please run the copyright headers tool in this project')
+                    process.exit(1)
+                } else {
+                    console.log('\x1b[32m\x1b[40mSUCCESS\x1b[49m - \x1b[36m All copyright headers present')
+                }
+            } else {
+                console.log('\x1b[32m\x1b[40mSUCCESS\x1b[49m - Copyright headers finished')
+            }
+        })
+
+    },
+    hasCopyrightHeader(data) {
+        if (data.indexOf('Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved.') >= 0) {
+            return true
+        }
+        return false
+    },
```

Comment:
This function can just be `return data.indexOf(COPYRIGHT_STRING) >= 0`.

It doesn't actually use `this`, so it could even move outside this object and become even simpler.